modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,106 @@
+[[clients-oracle]]
+= Registering {oracle} Clients
+
+This section contains information about registering traditional and Salt clients running {oracle} operating systems.
+
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{oracle} repository URLs are available from {scc}, but the packages and metadata are provided by Oracle, not by {suse}.
+{oracle} base media repositories are freely available for download from https://yum.oracle.com/.
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+
+== Channel and Repository Management
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
+* For {oracle} 7:
++
+From the {webui}, add [systemitem]``Oracle Linux 7 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux7-x86_64``.
++
+* For {oracle} 8:
++
+From the {webui}, add [systemitem]``Oracle Linux 8 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux8-x86_64``.
+. Synchronize the {productname} Server with the {SCC}.
+You can do this using the {webui}, or by running [command]``mgr-sync`` at the command prompt.
+. Add the new channel to your activation key.
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+[IMPORTANT]
+====
+For {oracle} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
+You will require packages from both channels.
+If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+====
+
+
+.Procedure: Adding Client Tools Channels
+. At the command prompt on the {productname} Server, as root, install the [package]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the {oracle} base, updates, and client channels, specifying the {oracle} version and architecture:
++
```

Comment:
This is not enough, we should add CLI calls for 6 and 7 as well, as they don't have the same chanels ad 8.

Also keep in mind there are NO oraclelinux8-uyuni-client, just as for SUSE manager the client tools for RES are to be used, for Uyuni the CentOS client tools are to be used:

So here are the channels for each case:
```
Oracle 6: oraclelinux6 centos6-uyuni-client
Oracle 7: oraclelinux7 centos7-uyuni-client
Oracle 8: oraclelinux8 oraclelinux8-appstream centos8-uyuni-client
```

Note: I just noticed https://www.uyuni-project.org/uyuni-docs/uyuni/client-configuration/clients-centos.html#_channel_and_repository_management does NOT mention the client tools. Will send a PR to fix it now.

Reply:
> This is not enough, we should add CLI calls for 6 and 7 as well, as they don't have the same chanels ad 8.
> 
> Also keep in mind there are NO oraclelinux8-uyuni-client, just as for SUSE manager the client tools for RES are to be used, for Uyuni the CentOS client tools are to be used:
> 
> So here are the channels for each case:
> 
> ```
> Oracle 6: oraclelinux6 centos6-uyuni-client
> Oracle 7: oraclelinux7 centos7-uyuni-client
> Oracle 8: oraclelinux8 oraclelinux8-appstream centos8-uyuni-client
> ```
> 
> Note: I just noticed https://www.uyuni-project.org/uyuni-docs/uyuni/client-configuration/clients-centos.html#_channel_and_repository_management does NOT mention the client tools. Will send a PR to fix it now.

Done, but is that correct that it's CentOS?

Reply:
Sorry, I am blind: `centos*-uyuni-client` is actually there ;-)